_posts/2020-11-26-torch-brain-segmentation/torch_brain_segmentation.Rmd

```diff
@@ -0,0 +1,705 @@
+---
+title: "Brain image segmentation with torch"
+description: > 
+  The need to segment images arises in various sciences and their applications, many of whom are vital to human (and animal) life. In this introductory post, we train a U-Net to mark lesioned regions on MRI brain scans.
```

Comment:
```suggestion
  The need to segment images arises in various sciences and their applications, many of which are vital to human (and animal) life. In this introductory post, we train a U-Net to mark lesioned regions on MRI brain scans.
```

Reply:
Since you're referring back to sciences and their applications and not a person 👍